chunks/tool-chrome/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG base
+FROM ${base}
+
+USER gitpod
+
+# Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
+ENV TRIGGER_REBUILD=1
+
+# chrome and basic render font
+RUN apt-get update \
+  && apt-get install -y apt-transport-https \
+  && curl -sSL https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add - \
+  && echo "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google-chrome.list \
+  && apt-get update && apt-get install -y google-chrome-stable --no-install-recommends \
```

Comment:
Awesome, we are super close now! Let me know what you think of the below suggestion? This will help this new chunk follow [our contribution guide](https://github.com/gitpod-io/workspace-images/blob/master/CONTRIBUTING.md#adding-a-chunk). I think you'll like number 8 ([here is supporting info](https://www.linuxuprising.com/2021/01/apt-key-is-deprecated-how-to-add.html)).

And, now that we're socializing this PR, we should add a number 9 (when and why to use `install-packages`).

For example, we're using apt 2.0, and it looks like [https is supported out of the box with 1.5+](https://unix.stackexchange.com/a/90229).

```shell
USER root
RUN curl -sSL https://dl.google.com/linux/linux_signing_key.pub | sudo gpg --dearmor -o /usr/share/keyrings/chrome-archive-keyring.gpg \
   && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/chrome-archive-keyring.gpg] https://dl.google.com/linux/chrome/deb/ \
    stable main" | sudo tee /etc/apt/sources.list.d/chrome.list > /dev/null \
    && apt update

USER gitpod
# you can append more package you with to install
# install-packages does many things for you, like update and clean-up afterwards
sudo install-packages google-chrome-stable

```

Reply:
after apply this changes, when I run `apt update`, I got 

```console
E: Conflicting values set for option Signed-By regarding source https://dl.google.com/linux/chrome/deb/ stable: /usr/share/keyrings/chrome-archive-keyring.gpg != 
E: The list of sources could not be read.
```

Reply:
@kylos101

Reply:
Hey @gengjiawen ,

This works better (I just tested).

```dockerfile
USER root

# Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
ENV TRIGGER_REBUILD=1

# chrome and basic render font
RUN curl -fsSL https://dl.google.com/linux/linux_signing_key.pub | sudo gpg --dearmor -o /usr/share/keyrings/chrome-archive-keyring.gpg \
   && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/chrome-archive-keyring.gpg] https://dl.google.com/linux/chrome/deb/ stable main" | sudo tee /etc/apt/sources.list.d/chrome.list > /dev/null \
    && apt update \
    && install-packages google-chrome-stable

USER gitpod

# misc deps for electron and puppeteer to run
RUN sudo install-packages \
    libasound2-dev libgtk-3-dev libnss3-dev \
    fonts-noto fonts-noto-cjk
```

Can you check, to assert it fits your needs?